ember/app/models/identity.js

```diff
@@ -1,14 +1,18 @@
-import Model, { attr, hasMany } from "@ember-data/model";
+import { attr, hasMany } from "@ember-data/model";
+import { LocalizedModel } from "ember-localized-model";
 
-export default class IdentityModel extends Model {
+export default class IdentityModel extends LocalizedModel {
```

Comment:
There is not really a point to extend from `LocalizedModel` if you dont use the decorator.

Reply:
As discussed. We'll clean that up when everything is final.